packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -162,6 +168,12 @@ export function getPropGetters<
     const { inputElement, maxLength = 512, ...rest } = providedProps || {};
     const activeItem = getActiveItem(store.getState());
 
+    const userAgent = props.environment.navigator?.userAgent;
+    const shouldFallbackKeyHint =
+      isSamsung(userAgent) && isAndroid(userAgent) && isChrome(userAgent);
```

Comment:
I wonder if for example we could create an input, change the keyhint and see if the value has has changed, in which case we keep it static for the real input?

Or maybe alternatively resetting the input value once the keyhint changes too?

Or never changing the keyhint?

Reply:
>is there another way than user agent to detect this behaviour?

Didn't find a way to detect the Predictive text feature, or disable it. The feature doesn't respect HTML specs (as in, won't turn off with `autocorrect="off"` so 🤷‍♀️)

>we could create an input, change the keyhint and see if the value has has changed, in which case we keep it static for the real input? Or maybe alternatively resetting the input value once the keyhint changes too?

I don't believe this would work because we'd need to "wait" for Predictive text to kick in and we don't control it, so this could result in delayed updates. I'm not sure updating the input value would change anything, as the Predictive Search feature happens after. From what we tried with @FabienMotte I don't remember we were successful at even delaying updates.

Also, if we create a manual fake input we'll need to make it work in environments such as React Native.

>Or never changing the keyhint?

This is the simplest choice, and the first one we considered. But if we do, we lose a feature because of a broken implementation on a single browser in a set of devices from a single constructor, which kind of sucks. The feature isn't vital though. It's a trade-off, we can discuss it together.

Reply:
> Or maybe alternatively resetting the input value once the keyhint changes too?

What about this option?